src/common/cluster-store.ts

```diff
@@ -73,20 +73,27 @@ export class ClusterStore extends BaseStore<ClusterStoreModel> {
       accessPropertiesByDotNotation: false, // To make dots safe in cluster context names
       migrations: migrations,
     });
-    if (ipcRenderer) {
-      ipcRenderer.on("cluster:state", (event, model: ClusterState) => {
-        this.applyWithoutSync(() => {
-          logger.silly(`[CLUSTER-STORE]: received push-state at ${location.host}`, model);
-          this.getById(model.id)?.updateModel(model);
-        })
-      })
-    }
   }
 
   @observable activeClusterId: ClusterId;
   @observable removedClusters = observable.map<ClusterId, Cluster>();
   @observable clusters = observable.map<ClusterId, Cluster>();
 
+  registerIpcListener() {
```

Comment:
Moved to a separate function so that we can call this later in bootstrap (registering IPC during "import" might cause problems later).

Reply:
Because initial `disconnected` previously was `falsy`